bin/build_mjml.sh

```diff
@@ -18,11 +18,11 @@ END
 MJML_REGEX="^[^_].+\.mjml$"
 
 if [ "$1" = 'help' -o "$1" = 'h' -o "$1" = '--help' ]; then
-  echo -n "$USAGE\n"
```

Comment:
@webknjaz Why do I need this `USAGE`? Just for little help if someone will want to use this script in other place without reading the code. I wasn't going to add this script to the `Makefile` while writing it, so I decided to add some help and now I have no idea why I should delete it :man_shrugging: